src/data/polynomial.lean

```diff
@@ -1854,6 +1854,16 @@ end
 lemma degree_eq_degree_of_associated (h : associated p q) : degree p = degree q :=
 let ⟨u, hu⟩ := h in by simp [hu.symm]
 
+lemma degree_eq_one_of_irreducible_of_root (hi : irreducible p) {x : α} (hx : is_root p x) :
```

Comment:
Given the order of the arguments, shouldn't this be called `degree_eq_one_of_root_of_irreducible`? Or maybe I'm confused...

Reply:
Irreducible is before the root in the statement and the name. `p->q->r` is called `r_of_p_of_q`, like `lt_of_le_of_lt`

Reply:
Ok, so I was confused...